### PR TITLE
Initial PR for new OCIO homepage

### PR DESCRIPTION
**Pull request by carolalynn22**
Submitting an initial PR for the new homepage for the V2 launch. Based off a Hugo template, will be deployed via github pages. Know it's a lot in one go, but it's been looked at along the way by @doug-walker on my fork.  Here are a few screen shots of the result:

Screen Shot 2020-08-09 at 11 18 56 PM
Screen Shot 2020-08-09 at 11 19 04 PM
Screen Shot 2020-08-09 at 11 19 24 PM

### CODE REVIEW

**Comment by michdolan** [comment]
@jmertic this all look good to you in terms of FOSS license, etc?

**Comment by Simran-B** [comment]
Looks very nice, great work Carol!

Quick reference:

- `cd` into the `docs/site/homepage` folder
- `hugo server --themesDir ../.. http://localhost:1313/` to run a local development server
- `hugo --themesDir ../..` to just build the website (result is written to `./public` dir)

**Comment by carolalynn22** [comment]
Yes thanks Simran! I need to go back through and update the README for sure. It's sad right now. :)

File: docs/site/homepage/data/en/supported_apps.yml [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    # portfolio item loop
+    - name : Silhouette
+      image : images/supported_apps/silhouette.png
+      categories : ["2D", "Roto", "Paint"]
+      content : Version 4.5 and up.
+      link : "http://www.silhouettefx.com/silhouette"
+
+    # portfolio item loop
+    - name : Adobe Substance
+      image : images/supported_apps/substance.png
```
Comment (michdolan) [inline docs/site/homepage/data/en/supported_apps.yml:162]: One small note, which can be addressed later. This Substance logo is the Substance Painter logo. Currently only Substance Designer has OCIO support. We could either swap in the Substance designer logo, or the more neutral Substance logo (no shape around the "S").

File: docs/site/LICENSE [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Attribution 3.0 Unported (CC BY 3.0)
```
Comment (jmertic) [inline docs/site/LICENSE:1]: Hey @carolalynn22 - the general guidance from the ASWF TAC is to leverage the CC-BY-40 license for non-code contributions ( http://creativecommons.org/licenses/by/4.0/ ). Not sure if there is a specific driver behind this choice or not - happy to discuss further if you'd like.

File: docs/site/README_theme.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# Easy Setup (Hugo + Netlify + Forestry)
```
Comment (jmertic) [inline docs/site/README_theme.md:1]: There seems to be several extraneous files in here that likely were part of the theme bundle, but don't appear to be required for operation. You may want to consider removing those.

File: docs/site/LICENSE (hunk 2 above)
Comment (michdolan) [reply docs/site/LICENSE:1]: I believe this license is from the theme.

Comment (jmertic) [reply docs/site/LICENSE:1]: Ah ok - in that case we should leave this as-is.

**Pull request merged**